exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/PlannerSettings.java

```diff
@@ -114,6 +114,28 @@
   public static final String UNIONALL_DISTRIBUTE_KEY = "planner.enable_unionall_distribute";
   public static final BooleanValidator UNIONALL_DISTRIBUTE = new BooleanValidator(UNIONALL_DISTRIBUTE_KEY, null);
 
+  // ------------------------------------------- Index planning related options BEGIN --------------------------------------------------------------
+  public static final String USE_SIMPLE_OPTIMIZER_KEY = "planner.use_simple_optimizer";
+  public static final BooleanValidator USE_SIMPLE_OPTIMIZER = new BooleanValidator(USE_SIMPLE_OPTIMIZER_KEY, null);
+  public static final BooleanValidator INDEX_PLANNING = new BooleanValidator("planner.enable_index_planning", null);
+  public static final BooleanValidator ENABLE_STATS = new BooleanValidator("planner.enable_statistics", null);
```

Comment:
Is it used for obtaining MapR-DB and Parquet table statistics?

Reply:
I will let @gparai elaborate on this but my understanding is that by keeping a general name such as `planner.enable_statistics` option we could potentially use it for any type of data source (including Parquet) once the broader stats capability is added to Drill.

Reply:
Yes, that was the intent - allow statistics use for any source with one option.